ansible_base/lib/dynamic_config/settings_logic.py

```diff
@@ -297,4 +299,28 @@ def get_dab_settings(
             if PRIMARY_CACHE not in caches or FALLBACK_CACHE not in caches:
                 raise RuntimeError(f'Cache definitions with the keys {PRIMARY_CACHE} and {FALLBACK_CACHE} must be defined when DABCacheWithFallback is used.')
 
+    if 'ansible_base.feature_flags' in installed_apps:
+        dab_data.setdefault('INSTALLED_APPS', copy(installed_apps))
+        dab_data['INSTALLED_APPS'].append('flags')
```

Comment:
Services already added `flags` so we need to avoid it being added twice
```suggestion
        dab_data.setdefault('INSTALLED_APPS', copy(installed_apps))
        if "flags" not in dab_data["INSTALLED_APPS"]:
            dab_data['INSTALLED_APPS'].append('flags')
```

Reply:
Great note! Will update